### ISSUE TITLE
Auspice narratives scroll-up bug

### ISSUE THREAD

**Issue by kairstenfay** [nextstrain/auspice#731]
Aupice narratives has a known bug where scrolling down then scrolling up sometimes causes the page to load a blank white screen and then redirect from `/narratives` to `/na`.

The surest way to recreate this is to go to [an example narrative](https://wnv-paper-narrative.herokuapp.com/narratives/map-bug) and scroll all the way to the bottom. Then click the link to go back to the top or manually scroll up. 

This issue happens sporadically and frequently.

**Comment by joverlee521** [comment]
Console error:
```
mapHelpers.js:218 Uncaught TypeError: Cannot read property 'demes' of null
    at updateVisibility (mapHelpers.js:218)
    at Map.maybeUpdateDemesAndTransmissions (map.js:354)
    at Map.maybeUpdateDemesAndTransmissions (react-hot-loader.development.js:661)
    at Map.componentWillReceiveProps (map.js:119)
    at callComponentWillReceiveProps (react-dom.development.js:11441)
    at updateClassInstance (react-dom.development.js:11651)
    at updateClassComponent (react-dom.development.js:14694)
    at beginWork (react-dom.development.js:15644)
    at performUnitOfWork (react-dom.development.js:19312)
    at workLoop (react-dom.development.js:19352)
updateVisibility @ mapHelpers.js:218
maybeUpdateDemesAndTransmissions @ map.js:354
maybeUpdateDemesAndTransmissions @ react-hot-loader.development.js:661
componentWillReceiveProps @ map.js:119
callComponentWillReceiveProps @ react-dom.development.js:11441
updateClassInstance @ react-dom.development.js:11651
updateClassComponent @ react-dom.development.js:14694
beginWork @ react-dom.development.js:15644
performUnitOfWork @ react-dom.development.js:19312
workLoop @ react-dom.development.js:19352
callCallback @ react-dom.development.js:149
invokeGuardedCallbackDev @ react-dom.development.js:199
invokeGuardedCallback @ react-dom.development.js:256
replayUnitOfWork @ react-dom.development.js:18578
renderRoot @ react-dom.development.js:19468
performWorkOnRoot @ react-dom.development.js:20342
performWork @ react-dom.development.js:20254
performSyncWork @ react-dom.development.js:20228
interactiveUpdates$1 @ react-dom.development.js:20495
interactiveUpdates @ react-dom.development.js:2170
dispatchInteractiveEvent @ react-dom.development.js:4882
react-dom.development.js:17117 The above error occurred in the <Map> component:
    in Map (created by Connect(Map))
    in Connect(Map) (created by Main)
    in div (created by Context.Consumer)
    in StyledComponent (created by styles__PanelsContainer)
    in styles__PanelsContainer (created by Main)
    in span (created by Main)
    in Main (created by Connect(Main))
    in Connect(Main) (created by MainComponentSwitch)
    in Suspense (created by MainComponentSwitch)
    in MainComponentSwitch (created by Connect(MainComponentSwitch))
    in Connect(MainComponentSwitch) (created by Root)
    in div (created by Root)
    in Root (created by HotExportedRoot)
    in AppContainer (created by HotExportedRoot)
    in HotExportedRoot
    in Provider

React will try to recreate this component tree from scratch using the error boundary you provided, AppContainer.
logCapturedError @ react-dom.development.js:17117
logError @ react-dom.development.js:17153
callback @ react-dom.development.js:18094
callCallback @ react-dom.development.js:16433
commitUpdateEffects @ react-dom.development.js:16472
commitUpdateQueue @ react-dom.development.js:16463
commitLifeCycles @ react-dom.development.js:17364
commitAllLifeCycles @ react-dom.development.js:18736
callCallback @ react-dom.development.js:149
invokeGuardedCallbackDev @ react-dom.development.js:199
invokeGuardedCallback @ react-dom.development.js:256
commitRoot @ react-dom.development.js:18948
(anonymous) @ react-dom.development.js:20418
unstable_runWithPriority @ scheduler.development.js:255
completeRoot @ react-dom.development.js:20417
performWorkOnRoot @ react-dom.development.js:20346
performWork @ react-dom.development.js:20254
performSyncWork @ react-dom.development.js:20228
interactiveUpdates$1 @ react-dom.development.js:20495
interactiveUpdates @ react-dom.development.js:2170
dispatchInteractiveEvent @ react-dom.development.js:4882
react-hot-loader.development.js:241 TypeError: Cannot read property 'demes' of null
    at updateVisibility (mapHelpers.js:218)
    at Map.maybeUpdateDemesAndTransmissions (map.js:354)
    at Map.maybeUpdateDemesAndTransmissions (react-hot-loader.development.js:661)
    at Map.componentWillReceiveProps (map.js:119)
    at callComponentWillReceiveProps (react-dom.development.js:11441)
    at updateClassInstance (react-dom.development.js:11651)
    at updateClassComponent (react-dom.development.js:14694)
    at beginWork (react-dom.development.js:15644)
    at performUnitOfWork (react-dom.development.js:19312)
    at workLoop (react-dom.development.js:19352)
error @ react-hot-loader.development.js:241
componentDidCatch @ react-hot-loader.development.js:2150
callback @ react-dom.development.js:18095
callCallback @ react-dom.development.js:16433
commitUpdateEffects @ react-dom.development.js:16472
commitUpdateQueue @ react-dom.development.js:16463
commitLifeCycles @ react-dom.development.js:17364
commitAllLifeCycles @ react-dom.development.js:18736
callCallback @ react-dom.development.js:149
invokeGuardedCallbackDev @ react-dom.development.js:199
invokeGuardedCallback @ react-dom.development.js:256
commitRoot @ react-dom.development.js:18948
(anonymous) @ react-dom.development.js:20418
unstable_runWithPriority @ scheduler.development.js:255
completeRoot @ react-dom.development.js:20417
performWorkOnRoot @ react-dom.development.js:20346
performWork @ react-dom.development.js:20254
performSyncWork @ react-dom.development.js:20228
interactiveUpdates$1 @ react-dom.development.js:20495
interactiveUpdates @ react-dom.development.js:2170
dispatchInteractiveEvent @ react-dom.development.js:4882
react-hot-loader.development.js:2157 Uncaught TypeError: Cannot read property 'demes' of null
    at updateVisibility (mapHelpers.js:218)
    at Map.maybeUpdateDemesAndTransmissions (map.js:354)
    at Map.maybeUpdateDemesAndTransmissions (react-hot-loader.development.js:661)
    at Map.componentWillReceiveProps (map.js:119)
    at callComponentWillReceiveProps (react-dom.development.js:11441)
    at updateClassInstance (react-dom.development.js:11651)
    at updateClassComponent (react-dom.development.js:14694)
    at beginWork (react-dom.development.js:15644)
    at performUnitOfWork (react-dom.development.js:19312)
    at workLoop (react-dom.development.js:19352)
updateVisibility @ mapHelpers.js:218
maybeUpdateDemesAndTransmissions @ map.js:354
maybeUpdateDemesAndTransmissions @ react-hot-loader.development.js:661
componentWillReceiveProps @ map.js:119
callComponentWillReceiveProps @ react-dom.development.js:11441
updateClassInstance @ react-dom.development.js:11651
updateClassComponent @ react-dom.development.js:14694
beginWork @ react-dom.development.js:15644
performUnitOfWork @ react-dom.development.js:19312
workLoop @ react-dom.development.js:19352
renderRoot @ react-dom.development.js:19435
performWorkOnRoot @ react-dom.development.js:20342
performWork @ react-dom.development.js:20254
performSyncWork @ react-dom.development.js:20228
interactiveUpdates$1 @ react-dom.development.js:20495
interactiveUpdates @ react-dom.development.js:2170
dispatchInteractiveEvent @ react-dom.development.js:4882
react-dom.development.js:17117 The above error occurred in the <AppContainer> component:
    in AppContainer (created by HotExportedRoot)
    in HotExportedRoot
    in Provider

Consider adding an error boundary to your tree to customize error handling behavior.
Visit https://fb.me/react-error-boundaries to learn more about error boundaries.
logCapturedError @ react-dom.development.js:17117
logError @ react-dom.development.js:17153
update.callback @ react-dom.development.js:18065
callCallback @ react-dom.development.js:16433
commitUpdateEffects @ react-dom.development.js:16472
commitUpdateQueue @ react-dom.development.js:16460
commitLifeCycles @ react-dom.development.js:17383
commitAllLifeCycles @ react-dom.development.js:18736
callCallback @ react-dom.development.js:149
invokeGuardedCallbackDev @ react-dom.development.js:199
invokeGuardedCallback @ react-dom.development.js:256
commitRoot @ react-dom.development.js:18948
(anonymous) @ react-dom.development.js:20418
unstable_runWithPriority @ scheduler.development.js:255
completeRoot @ react-dom.development.js:20417
performWorkOnRoot @ react-dom.development.js:20346
performWork @ react-dom.development.js:20254
performSyncWork @ react-dom.development.js:20228
interactiveUpdates$1 @ react-dom.development.js:20495
interactiveUpdates @ react-dom.development.js:2170
dispatchInteractiveEvent @ react-dom.development.js:4882
react-dom.development.js:20312 Uncaught TypeError: Cannot read property 'demes' of null
    at updateVisibility (mapHelpers.js:218)
    at Map.maybeUpdateDemesAndTransmissions (map.js:354)
    at Map.maybeUpdateDemesAndTransmissions (react-hot-loader.development.js:661)
    at Map.componentWillReceiveProps (map.js:119)
    at callComponentWillReceiveProps (react-dom.development.js:11441)
    at updateClassInstance (react-dom.development.js:11651)
    at updateClassComponent (react-dom.development.js:14694)
    at beginWork (react-dom.development.js:15644)
    at performUnitOfWork (react-dom.development.js:19312)
    at workLoop (react-dom.development.js:19352)
updateVisibility @ mapHelpers.js:218
maybeUpdateDemesAndTransmissions @ map.js:354
maybeUpdateDemesAndTransmissions @ react-hot-loader.development.js:661
componentWillReceiveProps @ map.js:119
callComponentWillReceiveProps @ react-dom.development.js:11441
updateClassInstance @ react-dom.development.js:11651
updateClassComponent @ react-dom.development.js:14694
beginWork @ react-dom.development.js:15644
performUnitOfWork @ react-dom.development.js:19312
workLoop @ react-dom.development.js:19352
renderRoot @ react-dom.development.js:19435
performWorkOnRoot @ react-dom.development.js:20342
performWork @ react-dom.development.js:20254
performSyncWork @ react-dom.development.js:20228
interactiveUpdates$1 @ react-dom.development.js:20495
interactiveUpdates @ react-dom.development.js:2170
dispatchInteractiveEvent @ react-dom.development.js:4882
react-dom.development.js:506 Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in the componentWillUnmount method.
    in ReactPageScroller (created by Narrative)
    in div (created by Narrative)
    in Narrative (created by Connect(Narrative))
    in Connect(Narrative) (created by Sidebar)
    in div (created by Context.Consumer)
    in StyledComponent (created by styles__SidebarContainer)
    in styles__SidebarContainer (created by Sidebar)
    in ThemeProvider (created by Sidebar)
    in Sidebar (created by Main)
    in span (created by Main)
    in Main (created by Connect(Main))
    in Connect(Main) (created by MainComponentSwitch)
    in Suspense (created by MainComponentSwitch)
    in MainComponentSwitch (created by Connect(MainComponentSwitch))
    in Connect(MainComponentSwitch) (created by Root)
    in div (created by Root)
    in Root (created by HotExportedRoot)
```

**Comment by jameshadfield** [comment]
Closed in #736
